### PR TITLE
DOC: Adds colab support to MNIST tutorial

### PR DESCRIPTION
**Pull request by thomasjpfan**
You can test it out here:

https://colab.research.google.com/github/thomasjpfan/skorch/blob/colab-mnist/notebooks/MNIST.ipynb

### CODE REVIEW

File: notebooks/MNIST.ipynb [hunk 1]
```diff
@@ -34,36 +41,26 @@
...
+    "from sklearn.model_selection import train_test_split\n",
+    "import numpy as np"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Loading Data\n",
+    "Using SciKit-Learns ```fetch_mldata``` to load MNIST data."
```
Comment (ottonemo) [inline notebooks/MNIST.ipynb]:
```suggestion
    "Using SciKit-Learns ```fetch_openml``` to load MNIST data."
```

Comment (benjamin-work) [reply notebooks/MNIST.ipynb]: ping @thomasjpfan

**Comment by benjamin-work** [comment]
Thanks!

**Comment by benjamin-work** [comment]
Ah, I just noticed that the entry in the CHANGES.md is missing, apart from that it's ready.

**Pull request merged**